.changeset/forty-crews-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/guide-cue': major
```

Comment:
Should this be patch?

Reply:
or since the multi-step tooltip is no longer using a portal that's a major change?